data/initrd/etc/rsyslog.d/logged_files.conf

```diff
@@ -7,3 +7,46 @@ input(type="imfile"
       Tag="yast2_logs"
       Severity="notice"
       Facility="local0")
+
+#linuxrc
+input(type="imfile"
+      File="/var/log/linuxrc.log"
+      Tag="linuxrc"
+      Severity="notice"
+      Facility="local0")
+
+
+#zypp history
+input(type="imfile"
+      File="/var/log/zypp/history"
+      Tag="zypp_history"
+      Severity="notice"
+      Facility="local0")
+
+#pk_backend_zypp
+input(type="imfile"
+      File="/var/log/pk_backend_zypp"
+      Tag="pk_backend_zypp"
+      Severity="notice"
+      Facility="local0")
+
+#zypp log
+input(type="imfile"
+      File="/var/log/zypper.log"
+      Tag="zypper"
+      Severity="notice"
+      Facility="local0")
+
+#pbl
+input(type="imfile"
+      File="/var/log/pbl.log"
+      Tag="pbl"
+      Severity="notice"
+      Facility="local0")
+
+#.packages.root
+input(type="imfile"
+      File="/.packages.root"
```

Comment:
It does not work. It turns also out that since this is a static file present in the ISO, that's how the user is supposed to retrieve it (eventually).

The commit was amended removing the entry.